src/styles/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 166px;
```

Comment:
should be 200px
hint: what is box-sizing for? let's try to apply it for all elements